tests/sles4sap/ipaddr2/configure.pm

```diff
@@ -8,8 +8,20 @@ use strict;
 use warnings;
 use Mojo::Base 'publiccloud::basetest';
 use testapi;
-use serial_terminal 'select_serial_terminal';
-use sles4sap::ipaddr2;
+use serial_terminal qw( select_serial_terminal );
+use sles4sap::ipaddr2 qw(
+  ipaddr2_bastion_key_accept
+  ipaddr2_bastion_pubip
+  ipaddr2_configure_web_server
+  ipaddr2_create_cluster
+  ipaddr2_deployment_logs
+  ipaddr2_destroy
+  ipaddr2_internal_key_accept
+  ipaddr2_internal_key_gen
+  ipaddr2_os_cloud_init_logs
+  ipaddr2_registeration_check
+  ipaddr2_registeration_set
```

Comment:
You're explicitly importing many functions. Wouldn't it be better to import all?
```
use sles4sap::ipaddr2;
```

Reply:
I do it automatically, I'm experimenting with `perlimport`. This tool always proposes and self calculate and explicit  import list. And as you pointed out this approach is not easy to maintain neither so meaningful when you have a module that mostly use all the API on another module. This is the case of lib/sles4sap/ipaddr2.pm importing lib/sles4sap/az_cli.pm. Here the tool suggest a too long list of many symbols. In that case I decided to ignore the perlimport suggestion there. Here the list was/is looking reasonable to me. I'd like to keep it and see how much effort is maintaining it.